config.js

```diff
@@ -12,7 +12,8 @@ const config = {
   moreInfoTitle: 'For more information about the issues follow the links: \n',
   tslintProjectWebsite: 'https://github.com/webschik/tslint-config-security',
   numFilesPerPage: 30,
-  numAnnotationsPerUpdate: 50
+  numAnnotationsPerUpdate: 50,
+  configFilePath: '.precaution.json'
```

Comment:
I do believe that YAML is a better human friendly config format than JSON.   I understand it's easier for a node.js app to consume JSON files, but YAML is better choices for users.

Reply:
Yea, maybe you are right. Because YAML gives you comments which are quite useful many times.
Also, it's just more readable and not so strict.

Reply:
I updated my pr with `.precaution.yaml` instead of `.precaution.json`.